src/pages/guides/using/idcs/idcs-events-data-stream-setup.md

```diff
@@ -0,0 +1,265 @@
+---
+title: Setting up IDCS Events Data Stream with Adobe I/O Events
+---
+
+# Setting up InDesign Cloud Services User Events Stream with Adobe I/O Events
+
+These instructions describe how to set up and get started using Adobe I/O Events for IDCS user driven job processing events.  You can use Adobe I/O for streaming IDCS job processing events such as the downloading of assets, engine processing, uploading of assets etc.
+
+## Introduction
+
+IDCS events provides all the job processing events that are shown in the sync status calls of InDesign cloud services api's. In addition to that events are realtime and more elaborate then sync api.
```

Comment:
This can be repharsed. As, we are not in sync with status call. In status call we are only having following events. "not-started" , "running", "completed" and "failed".

Reply:
Done